.rubocop.yml

```diff
@@ -8,7 +8,7 @@ Layout/IndentHeredoc:
   Enabled: false
 
 Metrics/AbcSize:
-  Max: 20
+  Enabled: false
```

Comment:
what does this do?

Reply:
rubocop has a metric called ABC http://wiki.c2.com/?AbcMetric

TBH i don't think it's very useful and we have a bunch of functions that disable it anyway